tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -59,7 +59,7 @@ spec:
         mkdir -p $RUNDIR
 
         default_pipeline_image="quay.io/redhat-isv/operator-pipelines-images:released"
-        request=$(jq -r '.sign.request // "hacbs-signing-pipeline"' ${DATA_FILE})
+        request=sign-image
```

Comment:
Do not merge

Reply:
So don't review it? Confused about this one since it is not a draft but you have do not merge. Is it ready for review?

Reply:
Yeah, I originally opened this as a draft. But it turned out that e2e won't run for a draft. And I wanted e2e to run to validate this change: https://gitlab.cee.redhat.com/signing/hacbs-signing-pipeline/-/merge_requests/19

And it worked. I saw all the pipelineruns in the stage internal-services cluster pass. So now I am confident that that one can be merged. And I will close this one.

Reply:
Sounds good. Just fyi you can have e2e run on a draft with `/test all` in a comment

Reply:
I see, thanks, didn't know that.

Reply:
(I will hold off on closing this until [the pipeline MR](https://gitlab.cee.redhat.com/signing/hacbs-signing-pipeline/-/merge_requests/19) is merged, in case I need to run the test again.)